e2e/deployers/discoveredapp.go

```diff
@@ -84,28 +84,20 @@ func (d DiscoveredApp) Undeploy(ctx types.Context) error {
 		return err
 	}
 
-	log.Infof("Deleting discovered apps on cluster %q", drpolicy.Spec.DRClusters[0])
```

Comment:
It can help to describe this change in the commit message (remove logs before operations when we log the operation in helper functions.